apollo-ios/Sources/ApolloAPI/DataDict.swift

```diff
@@ -179,11 +216,11 @@ extension Array: SelectionSetEntityValue where Element: SelectionSetEntityValue
       fatalError("\(Self.self) expected list of data for entity.")
     }
     self = data.map {
-#if swift(>=5.4)
+      if DataDict._AnyHashableCanBeCoerced {
         Element.init(_fieldData:$0)
```

Comment:
Looks like these lines need return statements now since its no longer technically a single line of code because of the `#if` check

Reply:
Yeah, thanks. just fixed that. Oddly it was still compiling and running fine on my machine with out them. I have no idea how.